eq-author-api/package.json

```diff
@@ -16,7 +16,9 @@
     "body-parser": "latest",
     "cheerio": "latest",
     "cors": "latest",
+    "deep-map": "latest",
```

Comment:
I'm unsure about this library.  It's untested and hasn't been updated for 3 years.  Can we implement this algorithm ourselves?

Reply:
Looks like it was updated 2 days ago to version 2? https://www.npmjs.com/package/deep-map

Reply:
https://www.npmjs.com/package/deep-map seems to be plenty of weekly downloads and was updated 2 days ago?

Reply:
My bad. Was looking at a different package: https://www.npmjs.com/package/deepmap